params/types/parity/parity.go

```diff
@@ -136,9 +137,9 @@ type ParityChainSpec struct {
 	Accounts map[common.UnprefixedAddress]*ParityChainSpecAccount `json:"accounts"`
 }
 
-func (c *ParityChainSpec) String() string {
+func (spec *ParityChainSpec) String() string {
 	cc := &ParityChainSpec{}
-	*cc = *c
+	*cc = *spec
```

Comment:
Unrelated; a product of a global replacement for consistent receiver name `spec`.